### PR TITLE
DBZ-6482 Document TimescaleDB SMT

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: documentation/modules/ROOT/pages/transformations/timescaledb.adoc [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+----
+<1> The name of the connector when registered with a Kafka Connect service.
+<2> The name of this PostgreSQL connector class.
+<3> The address of the TimescaleDB server.
+<4> The port number of the TimescaleDB server.
+<5> The name of the TimescaleDB user.
+<6> The password for the TimescaleDB.
+<7> The name of the TimescaleDB database to connect to.
+<8> The topic prefix for the TimescaleDB server/cluster, which forms a namespace and is used in all the names of the Kafka topics to which the connector writes, the Kafka Connect schema names, and the namespaces of the corresponding Avro schema when the Avro converter is used.
+<9> Indicates the usage of `pgoutput` logical decoding plug-in.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/transformations/timescaledb.adoc:195]:
```suggestion
<9> Indicates use of the `pgoutput` logical decoding plug-in.
```

File: documentation/modules/ROOT/pages/transformations/timescaledb.adoc [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+<10> A list of all schemas containing TimescaleDB physicial tables.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/transformations/timescaledb.adoc:196]:
```suggestion
<10> A list of all schemas that contain TimescaleDB physical tables.
```

File: documentation/modules/ROOT/pages/transformations/timescaledb.adoc [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+|No default
+|Name of the TimescaleDB database user for connecting to the TimescaleDB database server.
+
+|[[timescaledb-property-database-password]]<<timescaledb-property-database-password, `+database.password+`>>
+|No default
+|Password to use when connecting to the TimescaleDB database server.
+
+|[[timescaledb-property-database-dbname]]<<timescaledb-property-database-dbname, `+database.dbname+`>>
+|No default
+|The name of the TimescaleDB database from which to stream the changes.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/transformations/timescaledb.adoc:231]:
```suggestion
|The name of the TimescaleDB database from which to stream changes.
```

File: documentation/modules/ROOT/pages/transformations/timescaledb.adoc [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+|[[timescaledb-property-schema-list]]<<timescaledb-property-schema-list, `+schema.list+`>>
+|`_timescaledb_internal`
+|Comma-separated list schema names that contain TimescaleDB raw (internal) data tables.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/transformations/timescaledb.adoc:235]:
```suggestion
|Comma-separated list of schema names that contain TimescaleDB raw (internal) data tables.
```

**Review by roldanbob — CHANGES_REQUESTED** [review]

**Comment by jpechane** [comment]
@roldanbob Applied, thanks for the review!

**Pull request merged**